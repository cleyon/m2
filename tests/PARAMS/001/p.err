m2:p.m2:6:(dosubs) Text remains after scanning params
