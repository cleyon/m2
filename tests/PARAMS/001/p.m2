@@ @define __DEBUG__ 1
@@ @m2ctl dbg_params
@define foo Countdown: $11 .. $10 .. $9 .. $8 .. $7 .. $6 .. $5 .. $4 .. $3 .. $2 .. $1 .. $0 .. Lift-off!
@foo 1 2 3 4 5 6 7 8 9 10 11 12@
@foo{1}{2}{3}{4}{5}{6}{7}{8}{9}{10}{11}{12}@
@foo{aa}{bb}ccccc@
