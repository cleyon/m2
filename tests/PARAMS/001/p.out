Countdown: 11 .. 10 .. 9 .. 8 .. 7 .. 6 .. 5 .. 4 .. 3 .. 2 .. 1 .. foo .. Lift-off!
Countdown: 11 .. 10 .. 9 .. 8 .. 7 .. 6 .. 5 .. 4 .. 3 .. 2 .. 1 .. foo .. Lift-off!
